tests/unit/lms/views/application_instance_test.py

```diff
@@ -78,14 +82,24 @@ def test_it_sets_canvas_groups_enabled(
         pyramid_request.params["developer_key"] = developer_key
         pyramid_request.params["developer_secret"] = "test_developer_secret"
 
-        create_application_instance(pyramid_request)
+        application_instance = self.create_and_return_application_instance(
+            pyramid_request
+        )
 
-        application_instance = pyramid_request.db.query(ApplicationInstance).one()
         assert (
             bool(application_instance.settings.get("canvas", "groups_enabled"))
             == canvas_groups_enabled
         )
 
+    def create_and_return_application_instance(self, pyramid_request):
```

Comment:
There was a fair amount of repetition doing this in every test. Moved it to a function.